wagtail_localize/views/submit_translations.py

```diff
@@ -317,3 +318,38 @@ def get_success_message(self, locales):
             object=str(self.object),
             locales=locales,
         )
+
+
+class SubmitModelAdminTranslationView(SubmitTranslationView):
```

Comment:
imho this should live in `.modeladmin.views` since `modeladmin` is an optional sub-app

Reply:
Refactored modeladmin as an optional sub-app and moved the views and tests to modeladmin module.